core/services/ocr2/validate/validate.go

```diff
@@ -176,3 +179,29 @@ func validateOCR2VRFSpec(jsonConfig job.JSONConfig) error {
 	}
 	return nil
 }
+
+func validateOCR2KeeperSpec(jsonConfig job.JSONConfig) error {
+	if jsonConfig == nil {
+		return errors.New("pluginConfig is empty")
```

Comment:
We should use named errors whenever possible such that a package exports the error. 

Example:

```golang
var ErrConfigError = errors.New("configuration error")

// ... in this function
return fmt.Errorf("%w: plugin config is empty", ErrConfigError)
```

This allows for error type comparisons outside the package.

```golang
err := validate.CallSomeExportedFunction()

if errors.Is(err, validate.ErrConfigError) {
    // exit or catch
}
```